src/client/client.go

```diff
@@ -574,8 +577,24 @@ func NewInCluster(options ...Option) (*APIClient, error) {
 // NewInWorker constructs a new APIClient intended to be used from a worker
 // to talk to the sidecar pachd container
 func NewInWorker(options ...Option) (*APIClient, error) {
+	cfg, err := config.Read(false, true)
```

Comment:
Should we try and make an unauthenticated client if there's no config file in place? So we're not changing the existing behaviour.

Reply:
I think we should do this.

Reply:
This function is new for pachctl spouts, so there isn't really much existing behavior to speak of.